_posts/2015-03-03-Polls-API-Blueprint-Tutorial-Examples.md

```diff
@@ -0,0 +1,87 @@
+
+## Backend Implementation
+The Polls API comes with an [example implementation](http://github.com/apiaryio/polls-api) so you can use it as a real API:
+
+```bash
+$ curl https://polls.apiblueprint.org/questions/1
+```
+```json
+{
+    "question": "Favourite programming language?",
+    "published_at": "2014-11-11T08:40:51.620Z",
+    "url": "/questions/2",
+    "choices": [
+        {
+            "choice": "Swift",
+            "url": "/questions/1/choices/1",
+            "votes": 0
+        }, {
+            "choice": "Python",
+            "url": "/questions/1/choices/2",
+            "votes": 0
+        }
+    ]
+}
+```
+
+We've deployed an instance of this API and it's accessible for anyone to experiment with: `https://polls.apiblueprint.org/`. You can also deploy your own instance of the Polls API to Heroku with a single click:
+
+[![Deploy to Heroku](https://www.herokucdn.com/deploy/button.png)](https://heroku.com/deploy?template=https://github.com/apiaryio/polls-api)
+
+## Tested
+At Apiary we care about the whole **API lifecycle**. This is why we have created a testing tool for your blueprints and APIs – [Dredd](https://blog.apiary.io/2013/10/10/No-more-outdated-API-documentation).
+
+The Polls API implementation demonstrates the use of Dredd and as such its tested in a continuous integration as described in the [How To Test REST API with API Blueprint and Dredd](http://blog.apiary.io/2013/10/17/How-to-test-api-with-api-blueprint-and-dredd/) article.
+
+And yes, the badge bellow is live – just click on it to find out!
+
+[![Circle CI](https://circleci.com/gh/apiaryio/polls-api.svg?style=svg)](https://circleci.com/gh/apiaryio/polls-api)
+
+## Apiary Integration
+Last but not least we have changed the _default blueprint_ used when you create a new API in Apiary to the simple version of Polls API.
+
+For the first time in Apiary history and thanks to the Polls API implementation you can experiment with Apiary inspector (call debugger) features right with the default API. When you set documentation console to _"debugging proxy"_ or _"production"_ the calls you make are hitting the implementation endpoint instead of the Apiary mock server.
+
+<img width="599" src="/images/2015-03-03-apiary-integration.png" alt="Apiary Integration" />
+
+Note: The actual implementation of your API is set by the `HOST: http://polls.apiblueprint.org/` line on the top of tour blueprint.
+
+## API Design and Lifecycle
+We are striving to create the best tools to help everyone design beautiful API and guide it through its lifecycle.
+
+The new API Blueprint tutorial and its settings represents just that. From learning API Blueprint, choosing the right API design and prototyping to documenting, implementing, testing and using an API.
+
+Let us know how are we doing.
```

Comment:
Hope you enjoy the new tutorial and please let us know if you have any feedback or questions. We're always more than happy to help!